packages/website/docs/creating-a-renderer.md

```diff
@@ -1,14 +1,20 @@
 ---
 id: creating-a-renderer
-title: Creating a Renderer
+title: Creating a custom Renderer
```

Comment:
```suggestion
title: Creating a custom renderer
```

Reply:
I think there was a plan to fix the UI quirks later on cc @Shipow